docs/vault/upgrading.md

```diff
@@ -120,3 +120,100 @@ sudo systemctl start kintsugi-vault.service
 <!-- tabs:end -->
 
 OR start the [process manually](vault/installation?id=_5-start-the-vault-client).
+
+## Vault Auto-Updater
+
+Vaults can be run with [Runner](https://github.com/interlay/interbtc-clients/tree/master/runner) to automate updates. It reads on-chan release data (the `ClientsInfo::CurrentClientsRelease` storage map), downloads and runs new binaries as soon as chain storage is updated. The Runner restarts the vault on crash, thus also fulfilling the function of a Vault systemd service.
+
+### 1. Install from source
+
+#### 1.1 Install Rust
+
+```shell
+curl https://sh.rustup.rs -sSf | sh
+```
+
+#### 1.2 Build the Runner
+
+Clone the Runner code and build the binary:
+
+```shell
+git clone git@github.com:interlay/interbtc-clients.git
+cd interbtc-clients
+cargo build --bin runner
+```
+
+### 2. Start the Runner
+
+Move the runner binary into your `$PATH`.
+
+Pass Vault CLI arguments as positional arguments (preceeded by double dashes: `--`), after passing the command options of the runner executable. Example (on Kintsugi Testnet):
+
+```shell
+runner \
+    # Runner CLI arguments
+    --client-type vault \
+    --parachain-ws 'wss://api-dev-kintsugi.interlay.io:443/parachain' \
+    --download-path <CUSTOM_BINARY_DOWNLOAD_PATH, example: /opt/testnet/runner/> \
+    -- \
+    # Vault CLI arguments:
+    --bitcoin-rpc-url http://localhost:18332 \
+    --bitcoin-rpc-user rpcuser \
+    --bitcoin-rpc-pass rpcpassword \
+    --keyfile keyfile.json \
+    --keyname <INSERT_YOUR_KEYNAME, example: 0x0e5aabe5ff862d66bcba0912bf1b3d4364df0eeec0a8137704e2c16259486a71> \
+    --faucet-url 'https://api-dev-kintsugi.interlay.io/faucet' \
+    --auto-register=KSM=faucet \
+    --btc-parachain-url 'wss://api-dev-kintsugi.interlay.io:443/parachain'
+```
+
+### 3. [Optional] Start the Runner as a systemd service
+
+?> Some of the most common Linux systems support this approach (see [systemd](https://en.wikipedia.org/wiki/Systemd)).
+
+Download the systemd service file and a small helper script to install the service.
+
+#### **Testnet (Kintsugi)**
```

Comment:
Also add the Interlay testnet as an option.

Reply:
Added Kintsugi too